extensions/podman/src/extension.ts

```diff
@@ -307,6 +327,8 @@ async function registerProviderFor(provider: extensionApi.Provider, machineInfo:
     },
   };
 
+  containerProviderConnections.set(machineInfo.name, containerProviderConnection);
```

Comment:
hello, it looks like we never remove the field from the Map `containerProviderConnections`